kafka-ui-api/src/main/java/com/provectus/kafka/ui/cluster/service/ClusterService.java

```diff
@@ -62,6 +64,38 @@
         return kafkaService.createTopic(cluster, topicFormData);
     }
 
+    public Mono<ResponseEntity<ConsumerGroupDetails>> getConsumerGroupDetail(String clusterName, String consumerGroupId) {
+        KafkaCluster cluster = clustersStorage.getClusterByName(clusterName);
+        ConsumerGroupDetails result = new ConsumerGroupDetails();
+        result.setConsumerGroupId(consumerGroupId);
+        result.setConsumers(new ArrayList<>());
```

Comment:
Creating object before functions and then filling it in stream is very dirty!. Please do it in stream way